src/main/java/com/khutircraftubackend/product/ProductController.java

```diff
@@ -13,69 +15,80 @@
 import java.util.List;
 
 @RestController
-@RequestMapping("/products")
+@RequestMapping("/product")
```

Comment:
hey, you convinced me with plural, and i wrote a whole message about it with bunch of links, please check me previous comments :) 
so let't revert it back to "products" please, my apologies for confusion here. 
i've even created this jira ticket to rename other controllers where it's needed: SCRUM-79.

but, we still need an update here: `/v1/products`
and after this please update also `SecurityConfig` as well. 
thanks

Reply:
Ok.